microsoft-edge/webview2/release-notes.md

```diff
@@ -19,6 +19,65 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
 > [!NOTE]
 > WebView2 bug fixes are either Runtime or SDK specific.  
 
+## 1.0.955-prerelease
+
+Release Date: July 26, 2021  
+
+[NuGet package][NuGetGallery1.0.955-prerelease] \| Minimum Microsoft Edge version to load: 86.0.616.0 or newer \| Full API Compatibility: 93.0.955.0 or newer  
+
+### General  
+*   General WebView2 performance improvements.
+*   Added some ETW tracing support for WebView2.
+*   Removed Microsoft branding from edge://history.
+*   WebView2 has a new default Download UI.
+
+#### Experimental Features  
+*   Added [OpenTaskManagerWindow API][Webview2ReferenceWin32Icorewebview2experimental4ViewWebview210955PrereleaseOpentaskmanagerwindow] to launch a webview2 browser task manager.
+*   Added [NewWindowRequestedEventArgs][Webview2ReferenceWin32Icorewebview2experimentalnewwindowrequestedeventargsViewWebview210955PrereleaseGetName].
+
+#### Bug fixes  
+*   Fixed bug that broke the edge://downloads and edge://history pages. This change is Runtime-specific.
```

Comment:
Do we need to include this one here?
Work item: https://microsoft.visualstudio.com/Edge/_workitems/edit/32155177
PR: https://microsoft.visualstudio.com/DefaultCollection/Edge/_git/chromium.src/pullrequest/6095899

A short description for this would be: “Added support for virtual host name mapping to work with Service Worker.”.